internal/common/common_linux.go

```diff
@@ -68,7 +69,17 @@ func BootTimeWithContext(ctx context.Context) (uint64, error) {
 
 	filename := HostProc(statFile)
 	lines, err := ReadLines(filename)
-	if err != nil {
+	if os.IsPermission(err) {
+		var info syscall.Sysinfo_t
+		err := syscall.Sysinfo(&info)
+		if err != nil {
+			return 0, err
+		}
+
+		currentTime := time.Now().UnixNano() / int64(time.Second)
+		t := currentTime - int64(info.Uptime)
+		return uint64(t), nil
+	} else if err != nil {
```

Comment:
useless `else` when you `return` unconditionally in the preceding `if`

Reply:
```suggestion
	}
	if err != nil {
```